common/src/main/java/earth/terrarium/adastra/api/events/AdAstraEvents.java

```diff
@@ -108,6 +132,26 @@ static boolean post(ServerLevel level, LivingEntity entity) {
         }
     }
 
+    @FunctionalInterface
+    public interface EntityGravityEvent {
+        Optional<Float> tick(Entity entity, float gravity);
```

Comment:
Something like `getGravity` would make more sense than `tick`. This event is only ever posted when checking gravity, not based on any tick.

Reply:
changed